test/src/main/scala/scalarules/test/scala_import/BUILD

```diff
@@ -36,14 +28,6 @@ scala_specs2_junit_test(
     deps = [":guava_and_commons_lang"],
 )
 
-scala_specs2_junit_test(
-    name = "scala_import_exposes_file_jars",
-    size = "small",
-    srcs = ["ScalaImportExposesFileJarsTest.scala"],
-    suffixes = ["Test"],
-    deps = [":relate"],
-)
-
```

Comment:
this is about testing core bazel functionality introduced [here](https://github.com/mateuszkuta256/rules_scala/commit/efffa526a31724803532b46e659a7ea3240b9509)
thus I think it's not crucial to have such a test. Dropping it allows us to get rid of .jar file